eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/Utils.java

```diff
@@ -20,7 +20,9 @@
     )));
 
     private static final Set INVALID_FINAL_ANNOTATIONS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
-        "JsonProperty"
+        "JsonProperty",
+        "JsonAlias",
+        "JacksonXmlProperty"
```

Comment:
Not too sure why this has been modified?

Reply:
This is part of the rule.
We don't want to ask for any field with a Json annotation like the ones listed here to be final, since it might break serialization.

Reply:
I see - please add a comment to that effect to make it clear.

Reply:
yes, not a problem